src/org/ssgwt/client/validation/validators/RangeValidator.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2012 A24Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ssgwt.client.validation.validators;
+
+import org.ssgwt.client.i18n.SSDate;
+
+/**
+ * Range Validator.
+ *
+ * This class is used to validate that a number is within a range
+ *
+ * @author Deon De Wet <deon.dewet@a24group.com>
+ * @since  16 Sept 2020
+ */
+public class RangeValidator extends AbstractValidator implements ValidatorInterface<String> {
+
+    /**
+     * Name of the validator name.
+     */
+    private static final String VALIDATOR_NAME = "RangeValidator";
+
+    /**
+     * Default error message to use for validation
+     */
+    private static final String DEFAULT_VALIDATION_MESSAGE = "Number entered is not within the valid range";
```

Comment:
First, should this not be in strings somewhere or does ssGWT not have that?
Secondly, cant you maybe include the min and max here since this message wont be very useful. Maybe in the `getDefaultValidationMessage` you can add it at the back e,g `Number entered is not within the valid range (10-100)`

Reply:
I'm not using that message, it will default to that if no error message is passed in.